Assets/Scripts/Flashlight.cs

```diff
@@ -8,8 +8,6 @@ public class Flashlight : MonoBehaviour
     /// This code is used for the flashlight feature of this game. It works by turning a spotlight attached to the character on or off.
     /// </summary>
 
-
-
     //Flashlight variable.
     private Light flashLight;
 
```

Comment:
Extra space here unnecessary. Also Weird spacing in Update() makes it more difficult to read. Lastly, ensure that Start() and Update() are private.